text/0000-base-image-label.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Add base image SHA to stack metadata
+- Start Date: 2020-10-23)
+- Author(s): dumez-k
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Building off of this earlier [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0050-stack-metadata.md) outlining stack metadata,
+we would like to add a label to expose the stack base image SHA in the metadata of all stack-cnb images.
+
+# Motivation
+[motivation]: #motivation
+
+- Allows stack users and authors to easily see which base-image that each stack cnb image is based off of.
```

Comment:
I am assuming that by base-image we mean something like `ubuntu:bionic`?

Reply:
No actually this means the base image of the image that we provide AKA  `paketobuildpacks/build:full`